testsuite/features/secondary/srv_retracted_patches.feature

```diff
@@ -0,0 +1,115 @@
+# Copyright (c) 2015-2021 SUSE LLC
+# Licensed under the terms of the MIT license.
+
+@scope_retracted_patches
+Feature: The Retracted Patches
+
+  Scenario: Installed retracted package should show icon in the system packages list
+    Given I am authorized as "admin" with password "admin"
+    When I install package "rute-dummy-2.1-1.1.x86_64" on this "sle_minion"
```

Comment:
you created this package, right?

Reply:
mc did. But AFAIK, it's already present on the `head` environment.

Reply:
ok.